server/src/main/java/com/netflix/conductor/bootstrap/ModulesProvider.java

```diff
@@ -104,6 +113,34 @@ public ModulesProvider(Configuration configuration) {
             modules.add(new SwaggerModule());
         }
 
+        if (configuration.enableWorkflowExecutionLock()) {
+            Configuration.LOCKING_SERVER lockingServer;
+            try {
+                lockingServer = configuration.getLockingServer();
+            } catch (IllegalArgumentException ie) {
+                final String message = "Invalid locking server name: " + configuration.getLockingServerString()
+                        + ", supported values are: " + Arrays.toString(Configuration.LOCKING_SERVER.values());
+                logger.error(message);
+                throw new ProvisionException(message, ie);
+            }
+
+            switch (lockingServer) {
+                case REDIS:
+                    modules.add(new RedisLockModule());
+                    logger.info("Starting locking module using Redis cluster.");
+                    break;
+                case ZOOKEEPER:
+                    modules.add(new ZookeeperModule());
+                    logger.info("Starting locking module using Zookeeper cluster.");
+                    break;
+                default:
+                    break;
+            }
+        } else {
+            modules.add(new NoopLockModule());
+            logger.info("Starting locking module using Noop Lock. Are you sure with this setting?");
```

Comment:
The question `Are you sure with this setting?` is redundant because the user does not have a choice once this is executed. Consider removing this and instead log at `WARN` level.